ratis-server/src/main/java/org/apache/ratis/server/impl/RaftServerImpl.java

```diff
@@ -1513,6 +1513,10 @@ public CompletableFuture<AppendEntriesReplyProto> appendEntriesAsync(
     }
   }
 
+  public boolean initializing() {
+    return !startComplete.get();
+  }
```

Comment:
Let's check also the `lifeCycle`.  Also, remove `public`.
```java
  boolean isRunning() {
    return startComplete.get() && lifeCycle.getCurrentState() == State.RUNNING;
  }
```

Reply:
Ok, I will refer to this way.